src/main/java/org/scalameter/japi/annotation/scopeCtx.java

```diff
@@ -0,0 +1,26 @@
+package org.scalameter.japi.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+
+/** Points to a scope context.
```

Comment:
This comment is a bit unclear -- is this a shorthand for something that is a scope and a context.